### ISSUE TITLE
…the 'yt-dlp==2023.10.13' distribution was not found and is required by the application…

### ISSUE THREAD

**Issue by sapereaudete** [yt-dlp/yt-dlp#8430]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I'm running yt-dlp version **2023.10.13** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Germany

### Provide a description that is worded well enough to be understood

yt-dlp installed, de-installed, installed with both python3.6 and 3.11; 
Betriebssystem: openSUSE Leap 15.5
KDE-Plasma-Version: 5.27.4
KDE-Frameworks-Version: 5.103.0
Qt-Version: 5.15.8
Kernel-Version: 5.14.21-150500.55.31-default (64-bit)

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
yt-dlp -vU --write-description --no-post-overwrite --rm-cache-dir --write-thumbnail --no-warnings -f 18 https://www.youtube.com/watch?v=oB6wEaely-M
Traceback (most recent call last):
  File "/usr/bin/yt-dlp", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3251, in <module>
    @_call_aside
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3235, in _call_aside
    f(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3264, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 583, in _build_master
    ws.require(__requires__)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 900, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 786, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'yt-dlp==2023.10.13' distribution was not found and is required by the application
```

**Comment by Grub4K** [comment]
Python 3.6 is not supported anymore, so you should install using python 3.11 only. Remove the other, seemingly broken install and try again. I am assuming that you are trying to use the symlinked pip binary when you are calling `yt-dlp`? What happens if you run `python3.11 -m yt_dlp -vU`?

**Comment by sapereaudete** [comment]
$ python3.11 -m yt_dlp -vU
/usr/bin/python3.11: No module named yt_dlp

**Comment by Grub4K** [comment]
This seems like a packaging issue, which should instead be brought to the attention of the OpenSUSE repository maintainers

**Comment by richardstevenhack** [comment]
Reference the post Grub4K referred to above, I just did this on openSUSE Tumbleweed.

Ah, I just did this:

```
pipx install "yt-dlp>2023"
⚠  Note: yt-dlp was already on your PATH at /usr/bin/yt-dlp
  installed package yt-dlp 2023.10.13, installed using Python 3.11.5
  These apps are now globally available
    - yt-dlp
done! ✨ 🌟 ✨

```
Now it appears to run. The only difference from the previous time I tried the pipx install was the addition of ">2023."

I can now run the yt-dlp without calling it from python3 and directly referencing the /usr/bin/yt-dlp script. Apparently this new script in /usr/local/bin has changed the behavior somehow so as to make running the binary global differently than before. I need to test it with Media-Downloader but from the command line it appears to work now.
